imports/api/lib/methods.js

```diff
@@ -39,18 +24,19 @@ Meteor.methods({
 
   getPosts() {
     this.unblock();
-    const url = 'https://blog.keenethics.com/?format=json';
-    const result = Meteor.http.call('GET', url, {
-      params: {
-        timeout: 3000,
-      },
-      headers: {
-        cookie: 'sid=1:Ae8BWDRjWeqWqHqGt3Ik6om0Z8MYiyFSgKLtnEqcwb9GzRN5xVtNgay6WT1Yx5PU;' +
-                'uid=f82ad8efa94',
-        'content-type': 'application/json',
-      },
+    const parsingLib = new ParsingLib();
+    const getNewsData = new GetNewsData();
+    const mediumResult = parsingLib.parseMedium(getNewsData.getMediumJson());
+    const meetupResult = parsingLib.parseMeetup(getNewsData.getMeetupJson());
```

Comment:
Please, merge parsingLib and getNewsData. Let it have method like fetchNews that will return news for all the sources.